.npmrc

```diff
@@ -1 +1 @@
-package-lock=false
\ No newline at end of file
+package-lock=true
```

Comment:
I think we can just remove this file, this is the default. Never seen a project with a .npmrc file

Reply:
I have create a new branch `next` to land changes in.

In the case of this particular `.npmrc` file, I think it can be removed. Checking one in can be useful at times.